docs/sources/configure-client/grafana-alloy/receive_profiles.md

```diff
@@ -64,6 +64,8 @@ Check your specific language SDK documentation for the exact configuration optio
 
 The examples in this section provide samples you can use as a starting point for your own configurations. 
 
+Explore the [example](https://github.com/grafana/pyroscope/tree/main/examples/language-sdk-instrumentation/golang-push/rideshare-alloy) in our GitHub repository to learn how to configure Grafana Alloy to receive profiles from a Golang application instrumented with Pyroscope.
```

Comment:
Removing "our" in the sentence because we try not to use first person (we, I) in the docs except for things like release notes. 
```suggestion
Explore the [example](https://github.com/grafana/pyroscope/tree/main/examples/language-sdk-instrumentation/golang-push/rideshare-alloy) in the Pyroscope GitHub repository to learn how to configure Grafana Alloy to receive profiles from a Golang application instrumented with Pyroscope.
```